<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>168</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>120</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>168</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="2">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="minimumSize">
       <size>
        <width>700</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>700</width>
        <height>168</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::Box</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QPushButton" name="weeklyButton">
          <property name="text">
           <string>wöchentliche Abrechnung</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="3">
         <widget class="QComboBox" name="comboFahrer"/>
        </item>
        <item row="1" column="2">
         <widget class="QComboBox" name="comboFahrzeug"/>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="comboKW"/>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="backButton">
          <property name="text">
           <string>zurück</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QPushButton" name="loadButton">
          <property name="text">
           <string>Lade Daten</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuParameterauswahl">
    <property name="title">
     <string>Parameterauswahl</string>
    </property>
   </widget>
   <addaction name="menuParameterauswahl"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
